src/pages/Check/Check.styled.jsx

```diff
@@ -55,3 +55,15 @@ export const TextHandlerColorMobile = styled(Text)`
     font-size: 1.3rem;
   }
 `
+export const ImgSearch = styled.img`
+  object-fit: cover;
+  width: 357px;
+  z-index: -1;
```

Comment:
Do we really need z-index here?

Reply:
![chrome_SZ1q93YWAI](https://user-images.githubusercontent.com/71513679/174087040-96096b78-3fa7-4549-858d-45e49c665826.gif)

Reply:
Although it is because it is inside the modal. It could also be outside and use the state that conditions the modal.

Reply:
I think is because you used position: absolute, I think was not needed here

Reply:
@victor-heliomar please help here

Reply:
What about to add a bigger `z-index` to the modal instead of add a negative value to the image?